spec/version.dd

```diff
@@ -537,6 +538,127 @@ INT!(17) c;  // error, static assert trips
 	)
 	)
 
+$(H2 $(LNAME2 staticforeach, Static Foreach))
+
+$(GRAMMAR
+$(GNAME StaticForeach):
+    $(D static) $(GLINK2 statement, AggregateForeach)
+    $(D static) $(GLINK2 statement, RangeForeach)
+
+$(GNAME StaticForeachDeclaration):
+    $(GLINK StaticForeach) $(GLINK2 attribute, DeclarationBlock)
+    $(GLINK StaticForeach) $(D :) $(GLINK2 module, DeclDefs)$(OPT)
+
+$(GNAME StaticForeachStatement):
+    $(GLINK StaticForeach) $(GLINK2 statement, NoScopeNonEmptyStatement)
+)
+
+	$(P The aggregate/range bounds are evaluated at compile time and
+	turned into a sequence of compile-time entities by evaluating
+	corresponding code with a $(GLINK2 statement
```

Comment:
Missing comma at the end of the line (the link to `ForeachStatement` is broken).